### ISSUE TITLE
[EEv6.0.6] Image drag-drop file "Resolve Conflict" button goes to modal of 404 page.

### ISSUE THREAD

**Issue by PDXMan1** [ExpressionEngine/ExpressionEngine#1152]
Using an image upload field < within a grid field < within a fluid field < in the CP (file filed is assigned a specific directory).

If you drag a duplicate/existing file into the drop area, EE correctly offers a "Resolve Conflict" button within the drop area. Click that button and a front-end 404-page loads within a modal.

If you use the Upload New button vs. dragging the image, the resolve conflict form loads correctly.

PHP Version 7.3.28 / MySQL 5.7.34

URI: admin.php?/cp/publish/edit/entry/##

<img width="419" alt="Screen Shot 2021-06-18 at 2 08 19 PM" src="https://user-images.githubusercontent.com/6145078/122616533-66bb4c80-d03f-11eb-8a77-505f341984d7.png">

**Comment by Yulyaswan** [comment]
@PDXMan1 
I can't reproduce this issue. 
When I dragged an existing file (as you described) and clicked the `Resolve Conflict` button, I got a normal modal window
![New Entry About ExpressionEngine 2022-02-14 at 4 15 34 PM](https://user-images.githubusercontent.com/23382425/153880990-ad585ef1-ec92-47c1-81e3-1970cf7dfbf0.jpg)

Could you please give more details?

**Comment by PDXMan1** [comment]
That bug report is from 8 months ago. I'll need to find a client's site with an image upload field (that is assigned a specific directory) that's inside of a grid field that's within a fluid field. I think I remember the client's site that we found that issue in, but it's no longer in that same version of EE. Let me get it updated to the current build, test things, and I'll follow up with you. I'm curious though, did your test involve the conditions I listed above?
